core/src/autogluon/core/metrics/__init__.py

```diff
@@ -454,8 +454,11 @@ def local_pearsonr(y_true, y_pred):
                        greater_is_better=True)
 
 
-def rmse_func(y_true, y_pred):
-    return np.sqrt(((y_true - y_pred) ** 2).mean())
+def rmse_func(y_true, y_pred, **kwargs):
```

Comment:
Please add a simple unit test that calls this function with and without sample weights, verifying that it returns the correct score. You can refer to the existing metric unit tests for log_loss and roc_auc: https://github.com/autogluon/autogluon/blob/master/core/tests/unittests/metrics/test_classification_metrics.py#L130-L222